src/app/screens/Onboard/Intro/index.tsx

```diff
@@ -53,7 +53,7 @@ export default function Intro() {
         <Button
           onClick={() => navigate("/set-password")}
           type="button"
-          label="Get Started"
+          label={i18n.t("intro.get_started", namespaceI18n)}
```

Comment:
wondering if we should do something like this here: 

`i18n.t("intro.buttons.get_started", namespaceI18n)` 

somebody in weblate (or by looking at the json) probably does not have the context where this is used and it might then be unclear if this is for example a getting started text or rather just the Button "getting started" ?

Reply:
Done!